.vscode/tasks.json

```diff
@@ -137,6 +137,22 @@
                 ]
             }
         },
+        {
+            "label": "Build Lighting App (Tizen)",
```

Comment:
Remove this stand-alone build and add an entry to the 'examplePlatform' pick string below instead

Reply:
Done.